src/contributing.rst

```diff
@@ -192,7 +192,7 @@ The guidelines are in descending priority.
 
 #. Line length
 
-   * The maximum line length is ``80`` characters.
+   * The maximum line length is ``90`` characters.
```

Comment:
I wonder if we technically limit it at that in erlfmt or just have the 100 character limit.

Reply:
This limit is only for the documentation…

Reply:
See https://github.com/apache/couchdb-documentation/blob/f57d9b2ca93342da94365ae602e57635106efdf9/ext/linter.py#L206

Reply:
Ah, perfect. I confused that with the main repo